components/script/dom/bindings/codegen/CodegenRust.py

```diff
@@ -5532,8 +5533,10 @@ def getBody(self):
                 attrs = " | ".join(attrs)
             else:
                 attrs = "0"
-            fillDescriptor = ("desc.value = result_root.get();\n"
-                              "fill_property_descriptor(MutableHandle::from_raw(desc), proxy.get(), (%s) as u32);\n"
+            fillDescriptor = (#"desc.value_ = result_root.get();\n"
+                              "SetDataPropertyDescriptor(MutableHandle::from_raw(desc), result_root.handle(), (%s) as u32);\n"
+                              "*is_none = false;\n"
```

Comment:
TIL the `SetDataPropertyDescriptor` method! I'm wondering, would it be good to make the original `fill_property_descriptor` function to do these 2 lines so that we won't forget to set `is_none` to false in the future? 🤔?

Reply:
I implemented SetDataPropertyDesriptor in https://github.com/servo/mozjs/pull/318/commits/98e930f851dcb52b6c087d13abee87989b13b729#diff-2442927908394976e59adc5639205f7e248a8f4a3c21f483c9c1efb53e7ac1c0R1075, so that's not surprising. This is actually a good point!